torchtune/training/_activation_offloading.py

```diff
@@ -67,7 +67,7 @@ class OffloadActivations(saved_tensors_hooks):
     def __init__(
         self,
         use_pin_memory: bool = True,
-        use_streams: Optional[bool] = None,
+        use_streams: Optional[bool] = True,
         max_fwd_stash_size: int = 5,
         min_offload_size: int = 1024,
     ) -> None:
```

Comment:
let's remove the check below and make `use_streams: bool = True`

Reply:
are you referring to this `use_streams` check found in `__init__`? 

```python
        if use_streams is None:
            # Default to True if an acceptable torch is installed (later nightly/version or from source)
            self.use_streams = torch.__version__ >= "2.5.0.dev20240907"
        else:
            self.use_streams = use_streams
```

Reply:
I have noticed that `OffloadingActivations`  in `torchtune/training/_activation_offloading.py` still checks for torch version: `2.5.0.dev20240907`. Should this check also be removed? 

```python
        # for streaming
        if self.use_streams:
            if torch.__version__ < "2.5.0.dev20240907":
                raise RuntimeError(
                    "OffloadActivations with use_streams=True requires PyTorch 2.5.0.dev20240907 or later."
                )
```

Reply:
Yes, good catch. Let's remove that as well. I believe these may have been added after I put the issue up, or maybe I just missed it

Reply:
sounds good, will update all `2.5.0.dev20240907` checks in the file